### PR TITLE
Add inspyred

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/inspyred/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{%set name = "inspyred" %}
+{%set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/5e/f3/3d5f05b3be6336691277bcf088197caca789d0cf02c4a7fb2c3f21b021ab/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Please update based on the example.

File: recipes/inspyred/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  md5: ce51b599f5682ec526c97412f3ebaf45
+  patches:
+    - without-ez_setup.patch  # just use setuptools
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Seems reasonable. Could you please submit a PR upstream?

File: recipes/inspyred/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - 0001-Add-README.md.patch  # required from setup.py but missing from pypi archive
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Why do we need this? Also please raise an issue upstream to have `README.md` added to `MANIFEST.in`.

File: recipes/inspyred/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - inspyred
+
+about:
+  home: http://aarongarrett.github.io/inspyred/
+  license: MIT
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Please raise a PR/issue upstream to include the license file in `MANIFEST.in`.

File: recipes/inspyred/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+{%set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/5e/f3/3d5f05b3be6336691277bcf088197caca789d0cf02c4a7fb2c3f21b021ab/{{ name }}-{{ version }}.tar.gz
+  md5: ce51b599f5682ec526c97412f3ebaf45
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Please use the `sha256` checksum from [pypi.io]( https://pypi.org/ ).

File: recipes/inspyred/meta.yaml (hunk 4 above)
Comment (jschueller) [inline recipes/inspyred/meta.yaml]: ok

File: recipes/inspyred/meta.yaml (hunk 3 above)
Comment (jschueller) [inline recipes/inspyred/meta.yaml]: ok

File: recipes/inspyred/meta.yaml (hunk 2 above)
Comment (jschueller) [inline recipes/inspyred/meta.yaml]: ok

File: recipes/inspyred/meta.yaml (hunk 5 above)
Comment (jschueller) [inline recipes/inspyred/meta.yaml]: ok

File: recipes/inspyred/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/inspyred/meta.yaml]: ok

File: recipes/inspyred/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad472d9256aac445d6b5e2fb412118d76b25732bdbceec84fe6f16c343916957
+  patches:
+    - without-ez_setup.patch  # https://github.com/aarongarrett/inspyred/pull/8
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: xref: https://github.com/aarongarrett/inspyred/pull/8

File: recipes/inspyred/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    - 0001-Add-README.md.patch  # https://github.com/aarongarrett/inspyred/pull/9
```
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: xref: https://github.com/aarongarrett/inspyred/pull/9

File: recipes/inspyred/meta.yaml (hunk 4 above)
Comment (jakirkham) [inline recipes/inspyred/meta.yaml]: Done in PR ( https://github.com/aarongarrett/inspyred/pull/9 ).

**Pull request merged**